homework/english/Volha_Zharava/lecture1/task1.md

```diff
@@ -0,0 +1,46 @@
+Раскройте скобки, употребляя глаголы в Present Continuous или в Present Simple.
+Is your mum working today?
+Why are this boys talking now?
+Where does he live?
+Does their mother cook very well?
+Lili is not rich, she dosen't drive a Mercedes.
+Let´s go out. It isn't raining now.
+I have to go now. It is getting dark
+A young man usually stands at the window.
+Ann usually makes doll dresses herself.
+Does his wife have coffee at the moment?
+Let’s go for a walk. It is not raining.
+Look! The boys are coming home.
+Does Maila like Chinese food?
+My grandmother does not work. She is on pension.
+What is your brother watching now?
+Where do you live?
+Do you play the piano well?
+Where is their brother swimming now?
+What do they eat at breakfast?
+Are they swimming at the moment?
+Mike is not wearing a tie today.
+Maila does not watch TV every morning.
+Listen! Blanca is singing in the bathroom.
+Bea sometimes reads comics.
+
+Найдите ошибки и исправьте их.
+We do not go to school today.
```

Comment:
We do not go to school today - incorrect
 you should do the present continuous , not a present simple

Reply:
true. thx!!!